tests/nosetests/test_interferometry2/test_interferometry.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import sys
+import numpy as np
+import phoebe
+from astropy import units
+
+logger = phoebe.logger(clevel='INFO')
+#logger = phoebe.logger(clevel='DEBUG')
+
+b = phoebe.default_binary()
+
+times, u, v, wavelengths, vises, sigmas = np.loadtxt("Vis.dat", usecols=[0, 1, 2, 3, 5, 6], unpack=True)
```

Comment:
I think this is failing in the CI because it needs an absolute path.  Something like the following should work (the other test will need similar updates).

```suggestion
import os

dir = os.path.dirname(os.path.realpath(__file__))

logger = phoebe.logger(clevel='INFO')
#logger = phoebe.logger(clevel='DEBUG')

b = phoebe.default_binary()

path = os.path.join(dir, "Vis.dat")
times, u, v, wavelengths, vises, sigmas = np.loadtxt(path, usecols=[0, 1, 2, 3, 5, 6], unpack=True)
```

Reply:
Corrected. But 3 (unrelated) tests tend to fail...

Reply:
Hmm, it has something to do with .github/workfows, gfortran.